src/SampleUtterances.ts

```diff
@@ -160,8 +159,7 @@ export class SamplePhraseTest {
     private matchString: string;
 
     public constructor(public samplePhrase: SamplePhrase, private utterance: string) {
-        const cleanUtterance = utterance.replace(/[^0-9a-zA-Z ]/g, "");
```

Comment:
Maybe I left the unit tests out (which would be bad) but I think this was meant to deal with people putting in question marks and exclamation points and things like that. Will the code still work if someone adds special characters like those (which we want to ignore)?

Reply:
Makes sense, added the corresponding test and change the logic from allowing only a-z  and 0-9 characters to exclude symbols